pychunkedgraph/app/segmentation/v1/routes.py

```diff
@@ -244,12 +244,14 @@ def handle_leaves(table_id, node_id):
     return jsonify_with_kwargs(resp, int64_as_str=int64_as_str)
 
 
-@bp.route("/table/<table_id>/node/leaves_many", methods=["POST"])
```

Comment:
I am surprised. Was this used at all before?

Reply:
I doubt anyone is using this right now, so I decided to change the path as the "/node" part doesn't make that much sense as the function relates to many nodes